src/android/io/branch/BranchSDK.java

```diff
@@ -46,12 +46,12 @@ public BranchSDK()
     {
         this.activity = null;
         this.instance = null;
-        this.branchObj = null;
         this.callbackContext = null;
         this.onShareLinkDialogLaunched = null;
```

Comment:
Yes, I totally forgot about it. should be also referring to the instance.
Will make a follow-up commit.